### ISSUE TITLE
Conditional rules and success behavior are wrong after migrating to FF5

### ISSUE THREAD

**Issue by FrDH** [solspace/craft3-freeform#1488]
### What happened?

Hi,
We're trying to migrate all our sites from Freeform 4 to 5, but we noticed that the "conditional rules and success behavior setting aren't being migrated correctly.
After the update, the conditional rules are present, but they're different from before. The success behavior mostly was "Return to URL" , but after the update is always set to "Reload with success banner".
FYI: There is no error during the migration.

Here's what we've tried:
- update from the installed Freeform 4 to the latest Freeform 5
- update from the installed Freeform 4 to the lates Freeform 4 to the latest Freeform 5
- update from the installed Freeform 4 to the lates Freeform 4 to Freeform 5.0.0

All with the same result.
As we've got quite a lot of sites and some have pretty intense forms (heavily depending on conditional rules) on them, it would really help us a lot if we can sort this out. If there is anything we can do to help, please let me know. 

Also -a bit unrelated I guess-, after the update we need to re-label a lot of the submit buttons. What was "Versturen" before ("submit" in Dutch) now is set to "Submit". Looks like the custom value is translated. 

### How can we reproduce this?

1. Create a form that has conditional rules with Freeform 4
2. Update to Freeform 5
3. Check the conditional rules

### Freeform Edition

Pro

### Freeform Version

5 (multiple)

### Craft Version

4.9.7

### When did this issue start?

After upgrading from older Freeform version

### Previous Freeform Version

4 (multiple)

**Comment by kjmartens** [comment]
Sorry about this @FrDH. I'll have our developer check into this and we'll make some fixes. 🙂 

Due to the complexity of the changes in Freeform 5, there might be some limitations to the migration for conditional rules, but we'll see what we can do. Would you be able to elaborate on what you mean by "they're different from before"? 🙂

**Comment by FrDH** [comment]
Thanks, I think I tracked it down to rules that would show or hide a field based on an empty field:
```
[fieldName] [is not] ""
```
These are migrated to:
```
[fieldName] [is not equal to] ""
```
For some reason this messes up the behavior on the front end.
We manually changed these rules to use [is not empty]:
```
[fieldName] [is not empty]
```

**Comment by kjmartens** [comment]
Thanks for the additional information @FrDH.

We have just released Freeform 5.5.9, which includes fixes to the migration script and also now migrates submit button labels.

One issue we couldn't duplicate was the "Return to URL" submit behavior. It seemed to work correctly in our testing. I wonder if this might be because you have some really old forms that haven't updated to the new way we store the behavior options. In this case, saving each form while in Freeform 4 would likely solve this for the migration. 🙂

**Comment by FrDH** [comment]
Sorry for the late reply, we'll try 5.5.9 in the next round of updates.
Thanks, for the help.
